### PR TITLE
Harshad_Assignment_34_Done_45mins

### PR DESCRIPTION
**Pull request by HarshadBJun20**
-> Assignment 34 Done
-> Program 1 screenshot of output:
image

-> Program 2 screenshot of output:
image

Please review and comment.

### CODE REVIEW

File: src/harshadBJun20/assignment34/RemoveNameFromArrayList.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+		names.add("Maulik");
+		names.add("Techno");
+		names.add("Krishna");
+		names.add("Techno");
+		names.add("Credits");
+		names.add("Techno");
+		System.out.println("Given Array list:");
+		System.out.println(names);
+		System.out.println();
+		for(int index = 0;index<names.size();index++) {
```
Comment (AasthaJJun20) [inline src/harshadBJun20/assignment34/RemoveNameFromArrayList.java]: Your logic is correct but using while loop will reduce the iterations and if statement is also not required:
while(names.contains(nameToRemove))
				names.remove(nameToRemove);

File: src/harshadBJun20/assignment34/FindAndReplaceNameFromArrayList.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+Output : 
+Techno word index in given arrayList is 1,3,5 
+["Maulik","Credits" ,"Krishna", "Credits", "Credits", "Credits"]
+Total Credits words :- 4 */
+
+public class FindAndReplaceNameFromArrayList {
+	
+	void findAndReplaceName(ArrayList<String> namesList, String orgName, String replaceName) {
+		ArrayList<Integer> indexOfWord= new ArrayList<Integer>();
```
Comment (AasthaJJun20) [inline src/harshadBJun20/assignment34/FindAndReplaceNameFromArrayList.java:19]: Declaring just a counter variable, would also serve the purpose. But this is also fine.

File: src/harshadBJun20/assignment34/FindAndReplaceNameFromArrayList.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+		for(int index=0;index<namesList.size();index++) {
+				if(namesList.contains(orgName)) {
+					indexOfWord.add(namesList.indexOf(orgName));
+					namesList.set(namesList.indexOf(orgName), replaceName);
+				}				
+		}
+		System.out.println(orgName+" word index in given arrayList is "+indexOfWord);
+		System.out.println(namesList);
+		System.out.println("Total "+replaceName+" words :- "+(namesList.lastIndexOf(replaceName) - (namesList.indexOf(replaceName))));
```
Comment (AasthaJJun20) [inline src/harshadBJun20/assignment34/FindAndReplaceNameFromArrayList.java]: This is not correct logic. 
what if the below input is passed:
String[] names = {"Credits","Maulik","Techno" ,"Krishna", "Techno", "Credits", "Techno"};

**Review by AasthaJJun20 — CHANGES_REQUESTED** [review]
Please make the necessary changes for Program-2

File: src/harshadBJun20/assignment34/RemoveNameFromArrayList.java (hunk 1 above)
Comment (HarshadBJun20) [reply src/harshadBJun20/assignment34/RemoveNameFromArrayList.java]: ok Aastha Thanks. I will do necessary changes and update the PR.

File: src/harshadBJun20/assignment34/FindAndReplaceNameFromArrayList.java (hunk 2 above)
Comment (HarshadBJun20) [reply src/harshadBJun20/assignment34/FindAndReplaceNameFromArrayList.java:19]: ok Aastha Thanks. I will do necessary changes and update the PR.

File: src/harshadBJun20/assignment34/FindAndReplaceNameFromArrayList.java (hunk 3 above)
Comment (HarshadBJun20) [reply src/harshadBJun20/assignment34/FindAndReplaceNameFromArrayList.java]: ok Aastha Thanks. I will do necessary changes and update the PR.

File: src/harshadBJun20/assignment34/RemoveNameFromArrayList.java (hunk 1 above)
Comment (HarshadBJun20) [reply src/harshadBJun20/assignment34/RemoveNameFromArrayList.java]: Changes Done and updated PR as per review comment. Please review. Thanks

File: src/harshadBJun20/assignment34/FindAndReplaceNameFromArrayList.java (hunk 2 above)
Comment (HarshadBJun20) [reply src/harshadBJun20/assignment34/FindAndReplaceNameFromArrayList.java:19]: Changes Done and updated PR as per review comment. Please review. Thanks

File: src/harshadBJun20/assignment34/FindAndReplaceNameFromArrayList.java (hunk 3 above)
Comment (HarshadBJun20) [reply src/harshadBJun20/assignment34/FindAndReplaceNameFromArrayList.java]: Changes Done and updated PR as per review comment. Please review. Thanks

**Review by AasthaJJun20 — APPROVED** [review]
Code looks good to me.
Approving the PR.

**Pull request merged**